Assets/MixedRealityToolkit/_Core/Definitions/Utilities/AutoStartBehavior.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.Toolkit.Core.Definitions.Utilities
+{
+    /// <summary>
+    /// This enumeration gives the manager two different ways to handle the recognizer. Both will
+    /// set up the recognizer and add all keywords. The first causes the recognizer to start
+    /// immediately. The second allows the recognizer to be manually started at a later time.
+    /// </summary>
+    public enum AutoStartBehavior
+    {
+        /// <summary>
+        /// Automatically start the speech recognizer
```

Comment:
We should update the comment here. This should be generic to any recognizer.

Reply:
Updated and also moved feature to a new PR

Reply:
Thanks!